configs/env-example

```diff
@@ -111,6 +111,9 @@ FILE_UPLOAD_ALLOWED_EXTENSIONS=.pdf,.doc,.docx,.jpg,.png
 #SFAS Integration
 SFAS_RECEIVE_FOLDER=
 
+#Institution Integration
+IER_REQUEST_FOLDER=
```

Comment:
For all the ESDC integrations we have only one folder that receives different file names for each integration. Are we not going to follow the same for the "Institutions Integrations"?
`IER_REQUEST_FOLDER` is specific to the IER file only, right?